tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -32,8 +32,11 @@ using namespace std;
 using namespace std::chrono;
 using namespace std::filesystem;
 
-constexpr wstring_view badPath         = L"// ?? ?? ///// ?? ?? ? ////"sv;
-constexpr wstring_view nonexistentPath = L"C:/This/Path/Should/Not/Exist"sv;
+constexpr wstring_view badPath = L"// ?? ?? ///// ?? ?? ? ////"sv;
+path nonexistentPaths[]        = {

```

Comment:
Why is this global array modifiable? Could it instead be `constexpr wstring_view[]` with `path` constructed later on demand?

Reply:
It could be but then it's awkward to use / debug through because you step through path / wstring constructors constantly. I could make it const but it isn't used for anything constexpr here.